tests/test_schema_operations.py

```diff
@@ -37,24 +37,14 @@ def test_merge_doesnt_modify():
 
 
 def test_merge_conflicting():
-    """Test that merging conflicting values throws a NotImplementedError."""
-    schema_a = {"multipleOf": 3}
-    schema_b = {"multipleOf": 5}
+    """Test that merging conflicting values throws a SchemaIsFalse error."""
+    schema_a = {"hasDuplicates": True}
```

Comment:
`hasDuplicates` is an internal thing so probably not best for this example. And it turns out that `uniqueItems` is defined a bit oddly: https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.4.3, and doesn't actually have this behavior. `uniqueItems: false` is essentially ignored.